src/hydrabus/hydrabus_mode_spi.h

```diff
@@ -26,5 +26,7 @@ enum {
 	SPI_LSB_FIRST,
 };
 
-#endif /* _HYDRABUS_MODE_SPI_H_ */
+#define SPEED_NB (8)
+extern const uint32_t speeds[2][SPEED_NB];
```

Comment:
As it is required to export that it is better to rename it spi_speeds to avoid potential conflict in future